functional/tester/cluster_test.go

```diff
@@ -58,12 +58,13 @@ func Test_read(t *testing.T) {
 					InitialCluster:      "s1=https://127.0.0.1:1381,s2=https://127.0.0.1:2381,s3=https://127.0.0.1:3381",
 					InitialClusterState: "new",
 					InitialClusterToken: "tkn",
-					SnapshotCount:       10000,
```

Comment:
Sorry I did not follow why we need to change this?

Reply:
Without this: 
  go test ./functional/tester/...

fails. Please notice that 
  PASSES="unit" ./test -v 
nor
  PASSES="functional" ./test -v 

runs unit test in functional directory. 

https://github.com/etcd-io/etcd/blob/f395f82a757bd62086fc7b83be7b30f954d12662/test#L54
https://github.com/etcd-io/etcd/blob/f395f82a757bd62086fc7b83be7b30f954d12662/test#L217

If you ask why I'm setting 2000: 
It's because of this file: https://github.com/etcd-io/etcd/blob/f395f82a757bd62086fc7b83be7b30f954d12662/functional.yaml#L83

If you ask why I'm even touching the 'unused' test ? 
I'm pedantic about making sure all tests pass before I touch some code with serious changes ;)